ui/pages/device_complete.tsx

```diff
@@ -0,0 +1,15 @@
+import type { NextPage } from "next";
+import React from "react";
+import PageLayout from "../components/PageLayout";
+
+const DeviceComplete: NextPage = () => {
+  return (
+    <PageLayout title="Sign in successful">
+      <p>
+        Your device has been successfully registered. You can close this tab.
```

Comment:
I don't like the word registered for this, I think we should use `connected`

Reply:
Yes, registered has a specific meaning in the identity domain, I agree connected is better and will make that change shortly.